edm4hep.yaml

```diff
@@ -781,6 +781,19 @@ datatypes:
     OneToOneRelations:
       - edm4hep::Track track           // the corresponding track
 
+  #-------------  RecTof
+  edm4hep::RecTof:
+    Description: "Reconstructed TOF (time of flight) info."
+    Author: "EDM4hep authors"
+    Members:
+      - float time [ns]                      // time measurement
+      - std::array<float, 5> timeExp [ns]    // expected time for e(0),mu(1),pi(2),K(3),p(4)
```

Comment:
Should we make this a `VectorMember` to have a flexible number of hypotheses, instead of limiting ourselves to 5?

Reply:
The consensus from the discussion in the Key4hep meeting is that this is too much freedom.